app/grandchallenge/workstations/admin.py

```diff
@@ -36,6 +42,50 @@ class SessionHistoryAdmin(SimpleHistoryAdmin):
     ]
 
 
+class FeedbackAdmin(ModelAdmin):
+    readonly_fields = (
+        "user_comment",
+        "session",
+        "screenshot",
```

Comment:
Can we actually view the screenshot or does this need to be added to `serving`?

Reply:
Ah, sorry, you're right, this needs to be added to serving!